src/views/Home.vue

```diff
@@ -188,18 +184,16 @@ ion-card > ion-card-header {
   scroll-snap-type: x mandatory;
 }
 
-.scroller-card {
-  scroll-snap-align: start;
-}
-
 .scroller-item{
   flex-shrink: 0;
   position: relative;
   width: max-content;
   transform: translateX(calc(max(var(--page-width), 100vw)/2 - var(--page-width)/2));
+  /* set first block, second inline */
```

Comment:
I don't understand this comment.

Scroller item should snap align to the center

Reply:
Done

Reply:
Please be more descriptive with comments on PRs, what was done: did you remove something, did you update the comment?

Reply:
I have removed that comment sir and replaced `start end` property with `center`.

Reply:
thanks for the detailed comment